omegaconf/omegaconf.py

```diff
@@ -563,6 +563,8 @@ def _get_obj_type(c: Any) -> Optional[Type[Any]]:
         elif isinstance(c, DictConfig):
             if c._is_none():
                 return None
+            elif c._is_interpolation():
+                return None
```

Comment:
Please explain.

Reply:
Now that is_missing doesn't throw on resolution failure it doesn't matter this line. I'll remove it.